### PR TITLE
fix #175726: first step at getting plugins to work in master

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Bad thing happen when opening the Plugin Manager in an MSVC build: MuseScore crashes immediately. Not so in a MinGW build though?!?

**Comment by Jojo-Schmitz** [comment]
After #1000, #2000 and #3000 now also #4000 from me ;-)

**Comment by anatoly-os** [comment]
My local MSVC builds don't crash if I build release. Didn't check yet with debug or relwithdebinfo

File: share/plugins/helloqml/helloqml.qml [hunk 1]
```diff
@@ -26,10 +26,10 @@ MuseScore {
             while (segment) {
                 var element
                 element = segment.elementAt(0)
-                if (element && element.type == Element.CHORD) {
+                if (element() && element().type == Ms.CHORD) {
                     console.log(qsTr("    element"))
-                    console.log(element.beamMode)
-                    if (element.beamMode == BeamMode.NO)
+                    console.log(element().beamMode)
+                    if (element(9.beamMode == BeamMode.NO)
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply share/plugins/helloqml/helloqml.qml]: No, indeed. Will look at this tomorrow

**Pull request merged**